roles/common/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+rsyslog_certificate: "{{ lookup('file', inventory_dir+'/syslog_cert/rsyslogclient.crt') }}"
```

Comment:
All file included from the environment should go in the group_vars.

Add these three `syslog_*` variables to the group_vars template environment and describe their purpose and configuration there. For these https://github.com/OpenConext/Stepup-Deploy/blob/develop/environments/template/group_vars/all.yml would make the most sense because these are used by all hosts

Update the individual environments with the new variables.
Describe the change in the https://github.com/OpenConext/Stepup-Deploy/blob/branch-release-16/CHANGELOG so that deployers know what to do.

Does it make sense to generate these certs & key using https://github.com/OpenConext/Stepup-Deploy/blob/branch-release-16/environments/template/environment.conf ?